test/services/GetPenaltyDetailsServiceSpec.scala

```diff
@@ -199,5 +202,108 @@ class GetPenaltyDetailsServiceSpec extends SpecBase with LogCapturing {
       val result: Exception = intercept[Exception](await(service.getDataFromPenaltyServiceForVATCVRN("123456789")))
       result.getMessage shouldBe "Something has gone wrong."
     }
+
+    class SetUpWithMockConfig {
```

Comment:
Just thinking if this could be refactored into the existing 'Setup' class, could be:
```
class Setup(withRealConfig: Boolean = true) {
   val mockConfig: Configuration = mock(classOf[Configuration])
   val mockServicesConfig: ServicesConfig = mock(classOf[ServicesConfig])

   val mockAppConfig: AppConfig = new AppConfig(mockConfig, mockServicesConfig)
   val service = new GetPenaltyDetailsService(mockGetPenaltyDetailsConnector)(implicitly, appConfig = if(withRealConfig) appConfig else mockAppConfig)
   reset(mockGetPenaltyDetailsConnector)
}
```

Reply:
Done ✅